src/documents/tutorial-userInterfaceOptions/UsingImagesInContrastModes.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Using Images in Contrast Modes
+layout: default
+category: Tutorials
+---
+
+The User Interface Options component allows users to easily transform pages into one of several contrast versions. The results of this transformation are much more complete if any logos, images, and other graphics are also converted to match the contrast scheme.
+
+If you are integrating UI Options into your site, we highly recommend you create contrast versions of your site logo, and any other similarly static image. This page is divided into two sections:
+
+1. Creating contrast versions of your graphics using Photoshop (or similar image editing software), and
+2. Using CSS to switch between different contrast versions of the images.
+
+![A logo appearing properly with a yellow-on-black contrast theme](../images/tutorial-uio-good-bad-logos.png)
+
+## Converting Images With Transparency
+
+UI Options provides four high contrast themes:
+
+* black on white
+* white on black
+* black on yellow
+* yellow on black
+* light grey on dark grey
+
+These themes would require different versions of images and graphics to match. To do this you will need to use image editing software like Photoshop to change the colours of the images to match the different contrast themes.
+
+The simplest images to convert are ones that already have a transparent background. Creating contrast versions involves 'locking' the transparent pixels so they stay transparent and then painting over everything not-transparent with the desired colour.
+
+### Step 1 - Lock the transparent pixels
+
+In Photoshop, you can lock the transparent pixels in one of at least two ways:
+
+1. Type the '/' key, or
+2. In the "Layers" panel, click the 'lock pixels button (see image below).
+
+![Image of the Photoshop Layers panel with the mouse cursor hovering over the Lock transparent pixels button]
+
+### Step 2 - Fill the rest of the image with the desired colour
+
+In Photoshop, there are several ways to fill the image with a single colour:
+
+1. Type Alt + Delete on Mac, or Alt + Backspace on Windows to fill the image with the current foreground colour, or
```

Comment:
The keystrokes are getting lost in the text, can you make them stand out more. Maybe make them bold or put them on a separate line?